components/_patterns/00-base/layouts/grid/03-grid~divider-2.yml

```diff
@@ -1,6 +1,6 @@
 grid_label:
   "2-Column Divider Example"
-grid_modifiers:
+modifiers:
```

Comment:
If you agree with my last comment about variable inheritance, these should be reverted.